app/services/migrations/add_service_target_migrator.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Migrations
+  class AddServiceTargetMigrator
+    def self.call
+      new.run
+    end
+
+    delegate :query_service, to: :change_set_persister
+
+    def run
+      query_service.find_all_of_model(model: RasterResource).each do |raster_resource|
+        Wayfinder.for(raster_resource).file_sets.each do |file_set|
```

Comment:
We have a mosaic in production now with uncropped tiles, so I think we only want to migrate rasters where there's only one FileSet underneath the Raster.

Maybe something like:

```ruby
  file_sets = Wayfinder.for(raster_resource).file_sets
  if file_sets.length == 1
    # migrate
  end
```